packages/components/inputs/localized-text-input/src/localized-text-input.tsx

```diff
@@ -31,9 +28,130 @@ import {
   getLanguageLabelStyles,
 } from './localized-text-input.styles';
 
+type TEvent = {
+  target: {
+    name?: string;
+    value?: unknown;
+  };
+};
+
+type TLocalizedTextInputProps = {
+  id?: string;
+  name?: string;
+  autoComplete?: string;
+  // then input doesn't accept a "languages" prop, instead all possible
+  // languages have to exist (with empty or filled strings) on the value:
+  //   { en: 'foo', de: '', es: '' }
+  value: {
+    [key: string]: string;
+  };
+  /**
+   * Gets called when any input is changed. Is called with the change event of the changed input.
+   */
+  onChange?: (event: TEvent) => void;
+  /**
+   * Specifies which language will be shown in case the `LocalizedTextInput` is collapsed.
+   */
+  selectedLanguage: string;
+  /**
+   * Called when any field is blurred. Is called with the `event` of that field.
+   */
+  onBlur?: () => void;
```

Comment:
Is there a reason FocusEventHandler isn't used here instead?
```suggestion
  onBlur?: FocusEventHandler;
```